crates/node/src/command.rs

```diff
@@ -56,6 +56,7 @@ impl SubstrateCli for Cli {
 }
 
 fn set_dev_environment(cli: &mut Cli) {
+    println!("Setting up dev environment");
```

Comment:
debug log?

Reply:
okay, this was not supposed to exist at all, one of my debugging probably. I will delete it